doc/api/stream.md

```diff
@@ -633,22 +636,22 @@ within the `Readable` stream implementation.
 Specifically, at any given point in time, every `Readable` is in one of three
 possible states:
 
-* `readable.readableFlowing = null`
-* `readable.readableFlowing = false`
-* `readable.readableFlowing = true`
+* `readable.readableFlowing == null`
+* `readable.readableFlowing == false`
+* `readable.readableFlowing == true`
```

Comment:
Maybe switching to `===` makes sense, given the “flexible” nature of normal JS equality comparisons?